src/test/java/io/neonbee/test/endpoint/odata/ODataExpandEntityCollectionTest.java

```diff
@@ -0,0 +1,96 @@
+import static io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle.CHEESE_PRODUCT;
+import static io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle.PRODUCTS_ENTITY_SET_FQN;
+import static io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle.STEAK_PRODUCT;
+import static io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle.STREET_GLIDE_SPECIAL_PRODUCT;
+import static io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle.S_1000_RR_PRODUCT;
+import static io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle.addCategoryToProduct;
+import static io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle.getDeclaredEntityModel;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import io.neonbee.test.base.ODataEndpointTestBase;
+import io.neonbee.test.base.ODataRequest;
+import io.neonbee.test.endpoint.odata.verticle.NavPropsCategoriesEntityVerticle;
+import io.neonbee.test.endpoint.odata.verticle.NavPropsProductsEntityVerticle;
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.json.JsonObject;
+import io.vertx.junit5.Timeout;
+import io.vertx.junit5.VertxTestContext;
+
+public class ODataExpandEntityCollectionTest extends ODataEndpointTestBase {
+    @Override
+    protected List<Path> provideEntityModels() {
+        return List.of(getDeclaredEntityModel());
+    }
+
+    @BeforeEach
+    void setUp(VertxTestContext testContext) {
+        CompositeFuture
+                .all(deployVerticle(new NavPropsProductsEntityVerticle()),
+                        deployVerticle(new NavPropsCategoriesEntityVerticle()))
+                .onComplete(testContext.succeedingThenComplete());
+    }
+
+    @Test
+    @Timeout(value = 2, timeUnit = TimeUnit.SECONDS)
+    @DisplayName("Expand property 'category' in Products")
+    void testExpandCategoryInProdcuts(VertxTestContext testContext) {
+        ODataRequest oDataRequest = new ODataRequest(PRODUCTS_ENTITY_SET_FQN).setExpandQuery("category");
+        List<JsonObject> expected = List.of(addCategoryToProduct(STEAK_PRODUCT, FOOD_CATEGORY),
+                addCategoryToProduct(CHEESE_PRODUCT, FOOD_CATEGORY),
+                addCategoryToProduct(S_1000_RR_PRODUCT, MOTORCYCLE_CATEGORY),
+                addCategoryToProduct(STREET_GLIDE_SPECIAL_PRODUCT, MOTORCYCLE_CATEGORY));
+
+        assertODataEntitySetContainsExactly(requestOData(oDataRequest), expected, testContext)
+                .onComplete(testContext.succeedingThenComplete());
+    }
+
+    @Test
+    @Timeout(value = 2, timeUnit = TimeUnit.SECONDS)
+    @DisplayName("Do not expand property 'category' in Products")
+    void testDoNotExpandCategoryInProdcuts(VertxTestContext testContext) {
```

Comment:
typo: 
`testDoNotExpandCategoryInProdcuts ` to `testDoNotExpandCategoryInProducts `